otelcollector/deploy/addon-chart/azure-monitor-metrics-addon/templates/ama-metrics-daemonset.yaml

```diff
@@ -547,7 +547,7 @@ spec:
            - --token-server-listening-port=7777
            - --health-server-listening-port=9999
            - --restart-pod-waiting-minutes-on-broken-connection=240
-          image: "{{ .Values.AzureMonitorMetrics.AddonTokenAdapter.ImageRegistry }}{{ .Values.AzureMonitorMetrics.AddonTokenAdapter.ImageRepositoryWin }}:{{ .Values.AzureMonitorMetrics.AddonTokenAdapter.ImageTagWin }}"
+          image: "{{ $.Values.AzureMonitorMetrics.AddonTokenAdapter.ImageRegistry }}{{ $.Values.AzureMonitorMetrics.AddonTokenAdapter.ImageRepositoryWin }}:{{ $.Values.AzureMonitorMetrics.AddonTokenAdapter.ImageTagWin }}"
```

Comment:
Hi @gracewehner - Could you pls double check this renders correctly by doing a helm template for the aks daemonset chart too (bascically double check this works as expected outside the loop)?

Reply:
Thanks Rashmi, yes did this and everything gets substituted as expected